src/p2p/net_node.inl

```diff
@@ -714,6 +714,14 @@ namespace nodetool
   template<class t_payload_net_handler>
   bool node_server<t_payload_net_handler>::send_stop_signal()
   {
+    std::list<boost::uuids::uuid> connection_ids;
+    m_net_server.get_config_object().foreach_connection([&](const p2p_connection_context& cntxt) {
```

Comment:
I didn't comment on this last time, is it possible to close the connection in the `foreach_connection` callback? Or does that wreck the config object somehow?